### PR TITLE
Added the links in the youtube section sucessfully

### LINKED ISSUE #783: [Add] <Youtube >
### Site URL

https://www.youtube.com/@ApnaCollegeOfficial

### Description of the above provided site

Apna College is a popular YouTube channel dedicated to providing valuable content and resources to college students. The channel aims to support students in navigating various aspects of college life, including academics, career development, personal growth, and student well-being.

### Category

Youtube

### Sub Category

"Web Development"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Soumyajit2825**
Added 2 resources in the youtube section under web-development. 

This closes #783 and #782 

## Screenshots

Screenshot 2023-05-28 010059

### CODE REVIEW

**Review by ujjawaltyagii — APPROVED** [review]
Looks good to me 👍

**Review by CBID2 — APPROVED** [review]
Great job @Soumyajit2825! 😄 LGTM! 😄

**Review by rupali-codes — CHANGES_REQUESTED** [review]
@Soumyajit2825 could you please format the code?

**Comment by Soumyajit2825** [comment]
> @Soumyajit2825 could you please format the code?

Can you please brief me 
Which I have to format in the code?? @rupali-codes

**Comment by rupali-codes** [comment]
> > @Soumyajit2825 could you please format the code?
> 
> Can you please brief me Which I have to format in the code?? @rupali-codes

This file 
image

**Comment by Soumyajit2825** [comment]
@rupali-codes It's already formatted
You're viewing the soft-wrap view 
Switch to no-wrap view then you can see that the code is formatted already

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -158,5 +158,21 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "english"
+  },
+    {
```
Comment (rupali-codes) [inline database/youtube/web-development.json:162]:
```suggestion
   {
```

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**